src/Main/RestaurantsContainer/RestaurantsContainer.js

```diff
@@ -3,7 +3,7 @@ import './RestaurantsContainer.scss'
 
 const RestaurantsContainer = () => {
 
-    const [restaurants, setRestaurants] = useState([])
+    const [restaurants, setRestaurants] = useState(null)
 
     useEffect(() => {
         fetch('http://localhost:8080/restaurants')
```

Comment:
I thought you were gonna do response.ok etc but won't push it if you don't want to (was Rayna talking about Loading... rather than error handling?)

Reply:
All good as discussed